Pipfile

```diff
@@ -18,6 +18,8 @@ celery = "*"
 django-celery-results = "*"
 redis = "*"
 "discord.py" = "*"
+django-stubs = "*"
+celery-types = "*"
```

Comment:
so after adding these 2 dependencies, what did you run to find the unused imports and local variables? is it possible to add this as a precommit check?

Reply:
They were surfaced by the pyright LSP (I use neovim). I think VSCode will show these too but since I don't use it I can't say any more.